.travis.yml

```diff
@@ -28,19 +28,19 @@ jobs:
     - stage: "Unit tests"
       compiler: gcc
       name: "gcc"
-      if: NOT type = cron
+      if: branch = master AND NOT type = pull_request
```

Comment:
Is `NOT TYPE` correct here?

Reply:
Thanks. Should be corrected now to match the others in the "Unit Test" stage.